roles/ceph-common/defaults/main.yml

```diff
@@ -105,3 +105,25 @@ ceph_common:
   osd_disk_thread_ioprio_priority: 0
   osd_scrub_chunk_max: 5
   osd_deep_scrub_stride: 1048576
+
+  logs:
+    - paths:
+        - /var/log/ceph/ceph-osd*.log
+      fields:
+        type: openstack
+        tags: ceph,ceph-osd
+    - paths:
+        - /var/log/ceph/ceph-audit.log
+      fields:
+        type: openstack
+        tags: ceph,ceph-audit

```

Comment:
If this should be stored longer term, this should also have its own `audit` tag as Logstash does key off of that in where logs are sent for duplicated storage.